### ISSUE TITLE
Accessibility - 1010EZ: <title> element not informative because it doesn't change

### ISSUE THREAD

**Issue by davidakennedy** [department-of-veterans-affairs/va.gov-team#56834]
**Issue description**

The `<title>` element on all the pages is "Apply for VA health care". It never changes, so it isn't informative both to people who use screen readers and other assistive technology and also sighted users. It's tied to the h1 of the page, which is a good thing, but the h1 also never changes. See #56807. [See also this conversation in Slack](https://dsva.slack.com/archives/CMJ2V70UV/p1681336599975209).

![Birth information page on 1010EZ with red arrows pointing to h1 element on page and in the title element in HTML](https://images.zenhubusercontent.com/628e7b9093a9366153cb221c/e479dcdb-5b27-4836-8065-9f6def263958)

**Remediation guidance**

- All questions need to have unique <title> elements, and should continue to be tied to the h1.
- Probably dependent [on this design system issue](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1538).

**Does this impact visual design?**

No, but based on being tied to the h1 would require lots of input from Content.

**Relevant standards**

[WCAG 2.4.2 Page Titled](https://www.w3.org/TR/WCAG21/#page-titled)

**Comment by hdjustice** [comment]
Dependent on Design System team to make overarching change.  May be related to the work from the Forms team (currently testing an option).  Potentially not corrected by change, we will need to test.  
Revisit when Design System change is available
